html/changelogs/lohikar-lighting.yml

```diff
@@ -1,7 +1,7 @@
 author: Lohikar
 delete-after: True
 changes: 
```

Comment:
this stuff doesnt seem to have anything to do with doafters. I think you got some extraneous changelogs